databases/week3/main.sql

```diff
@@ -0,0 +1,254 @@
+USE dbweek3HW;
```

Comment:
would also include the create database statement in this file.

Reply:
ok,done.